src/auth/auth.go

```diff
@@ -0,0 +1,99 @@
+package auth
+
+import (
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+	"github.com/jinzhu/gorm"
+
+	"github.com/json9512/mediumclone-backendwithgo/src/users"
+)
+
+type loginCredentials struct {
+	Email    string `json:"email"`
+	Password string `json:"password"`
+}
+
+type errorResponse struct {
+	Msg string `json:"message"`
+}
+
+// AddRoutes adds HTTP Methods for the /posts endpoint
+func AddRoutes(router *gin.Engine, db *gorm.DB) {
+	router.POST("/login", func(c *gin.Context) {
+		var userCredentials loginCredentials
+		if err := c.BindJSON(&userCredentials); err != nil {
```

Comment:
그냥 이렇게 네이밍해도 될듯. credential.Email, credential.Password로 접근가능하니 단수가 나을듯
```suggestion
		var credential loginCredential
		if err := c.BindJSON(&credential); err != nil {
```